### ISSUE TITLE
Decision process for accessibility support

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#497]
During last week's joint meeting with the ACT-Rules CG we discussed how to handle accessibility support of test cases. During that call we decided that the ACT TF was going to come up with a process by which rule writers can find out if test cases with known accessibility support issues should pass or fail.
